subworkflows/local/utils_nfcore_metatdenovo_pipeline/main.nf

```diff
@@ -32,6 +32,7 @@ workflow PIPELINE_INITIALISATION {
     nextflow_cli_args //   array: List of positional nextflow CLI args
     outdir            //  string: The output directory where the results will be saved
     input             //  string: Path to input samplesheet
+    diamond_dbs       //  string: Path to csv file with Diamond taxonomy dbs
```

Comment:
is this path now mandatory? or optional?

Reply:
Optional, did I miss something? I can run the pipeline without specifying this so it seems to work, but it should of course be clear. Do you suggest I update the comment?

Reply:
Yes, I think it's good that we said both in comments and documentation (TBD)